river/metrics/prevalence_threshold.py

```diff
@@ -0,0 +1,95 @@
+import math
+
+from river import metrics
+
+__all__ = ["PrevalenceThreshold"]
+
+
+class PrevalenceThreshold(metrics.BinaryMetric):
+    r"""Prevalence Threshold (PT).
+
+    The relationship between a positive predicted value and its target prevalence
+    is propotional - though not linear in all but a special case. In consequence,
+    there is a point of local extrema and maximum curvature defined only as a function
+    of the sensitivity and specificity beyond which the rate of change of a test's positive
+    predictive value drops at a differential pace relative to the disease prevalence.
+    Using differential equations, this point was first defined by Balayla et al. [^1] and
+    is termed the **prevalence threshold** (\phi_e).
+
+    The equation for the prevalence threshold [^2] is given by the following formula
```

Comment:
Sounds redundant.

Reply:
I think the paragraph is necessary to explain the existence of the metric. Otherwise, it would be a bit difficult to understand how it will be applied.